pom.xml

```diff
@@ -1557,10 +1557,20 @@
                                 <rules>
                                     <bannedDependencies>
                                         <excludes>
-                                            <exclude>log4j:log4j</exclude>
-                                            <exclude>org.slf4j:slf4j-log4j12</exclude>
-                                            <exclude>ch.qos.logback:logback-classic</exclude>
+                                            <!-- we redirect logging to log4j2, so we should have those bridges instead -->
+                                            <!-- commons-logging is allowed because it natively sends to log4j2 or slf4j -->
+                                            <exclude>ch.qos.logback:*</exclude>
```

Comment:
Seems like the expectation is that slf4j-api has zero dependencies.  Wonder if adding something like `<exclude>*:*</exclude>` here as the only exclude would still build and would catch if slf4j-api ever did add any dep.

Reply:
Adding `<exclude>*:*</exclude` bans all dependencies. 
 
Regarding slf4j adding a dependency, I think that's covered by the exclude line on 1568 `<exclude>org.slf4j:*</exclude>` and then the individual include line for only slf4j-api on line 1572 `<include>org.slf4j:slf4j-api</include>`

Reply:
ah, I thought this section was related to just the sl4j-api but its way more than that.  Need to expand the diff.